installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -3444,4 +3444,9 @@ INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `def
 INSERT IGNORE INTO `cf_PolicyParams` (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('AWSRdsUnencrypted_version-1_AwsRdsUnencrypted','policyKey','check-for-rds-db-unencrypted','','false','false','false','','');
 
-UPDATE cf_PolicyTable SET severity ='critical' WHERE policyId IN ('Enable_HTTPS_For_LoadBalancer','Enable_transparent_data_encryption_for_sql_db');
\ No newline at end of file
+UPDATE cf_PolicyTable SET severity ='critical' WHERE policyId IN ('Enable_HTTPS_For_LoadBalancer','Enable_transparent_data_encryption_for_sql_db');
+
+INSERT IGNORE INTO cf_PolicyParams (policyID, paramKey, paramValue, defaultVal, isEdit, isMandatory, encrypt, displayName, description) VALUES ('AWSRdsPublicInstances_version-1_AwsRdsPublicAccess_rdsdb','cidrIp','0.0.0.0/0','','false','false','false','','');
+INSERT IGNORE INTO cf_PolicyParams (policyID, paramKey, paramValue, defaultVal, isEdit, isMandatory, encrypt, displayName, description) VALUES ('AWSRdsPublicInstances_version-1_AwsRdsPublicAccess_rdsdb','cidripv6','::/0','','false','false','false','','');
+INSERT IGNORE INTO cf_PolicyParams (policyID, paramKey, paramValue, defaultVal, isEdit, isMandatory, encrypt, displayName, description) VALUES ('AWSRdsPublicInstances_version-1_AwsRdsPublicAccess_rdsdb','internetGateWay','igw','','false','false','false','','');
+INSERT IGNORE INTO cf_PolicyParams (policyID, paramKey, paramValue, defaultVal, isEdit, isMandatory, encrypt, displayName, description) VALUES ('enable_aws_config_service','defaultregion','us-east-1','','false','false','false','','');
```

Comment:
Also replace `INSERT IGNORE` with `INSERT ... ON DUPLICATE KEY UPDATE`
https://dev.mysql.com/doc/refman/8.2/en/insert-on-duplicate.html 
`INSERT IGNORE` hides everything, but we meed to know if something broken

Reply:
@mkinchin-paladin @kushZemoso we can not update the parameter that users can update in each redeploy. Their values will override each time we redeploy. For such things, it is safe to use `insert ignore` so that we don't overwrite their data.

Reply:
@santhosh-challa What if you let's say made a mistake, mistyped something in your query. How it will appear in logs? In action?

Reply:
Ideally, we need to have migration scripts for this. Till then we need to double-check internally for any user updateable fields. As these are non-editable params we can use `INSERT ... ON DUPLICATE KEY UPDATE`. The only downside is that it updates the same value in each redeploy.  We can live with it till we have migration scripts in place.

Reply:
@mkinchin-paladin  @santhosh-challa  done with the changes pls review it.